flink-table/flink-sql-client/src/test/java/org/apache/flink/table/client/cli/CliClientTest.java

```diff
@@ -113,15 +113,9 @@ public void testSqlCompletion() throws IOException {
         verifySqlCompletion(
                 "show t ", 6, Collections.emptyList(), Collections.singletonList("SET"));
         verifySqlCompletion(
-                "show",
-                7,
-                Collections.singletonList("MODULES;"),
-                Collections.singletonList("QUIT;"));
+                "show", 7, Collections.singletonList("HintA"), Collections.singletonList("QUIT;"));
```

Comment:
We may not need this test any more, as the next one looks the same with this one.